packaging/linux/post_install.sh

```diff
@@ -189,6 +187,10 @@ else
   chmod a-s "$krbin"
 fi
 
+# Restart services before restarting redirector manually
+# so we know if it was already restarted via systemd.
+safe_restart_systemd_services
```

Comment:
There's still a behavior change from the last release, in that this is run before make_mountpoint at the end. I think it should be fine since if we're restarting keybase, the mountpoint should already exist.